requirements.in

```diff
@@ -32,6 +32,7 @@ isort==5.6.4
 ortools==9.4.1874
 psycopg2==2.8.6
 pydot==1.4.2
+pytest==7.1.3
```

Comment:
We should unpin the version - also i'm pondering the necessity of having pytest as hard requirement when we have pytest-django => obviously if some version clitches between two of those then it might be needed. :thinking:  You can upgrade a package to requirements.txt with pip-tools like; `pip-compile -P <package>`

Reply:
Yeah, valid point. The only reason I updated the package was the dependabot alert. In practice the alert is not that big of a deal because we are not using Subversion so we can ignore it too and hope that the existing dependencies get updated and decide to use newer version of pytest.

I can remove the pin, but what do you think, should we even do this or just live with the alert? ☺️

Reply:
I'd say lets update nevertheless :tada:

Reply:
![Cool beans!](https://media.giphy.com/media/cmd9kOHzBE7mftKkaI/giphy.gif)

I just remove the ping then 👍